sorc/build_gfs_util.sh

```diff
@@ -14,6 +14,12 @@ if [ ! -d "./mkgfsawps.fd" ]; then
    exit
 fi 
 
+if [ $target = jet ]; then
+   echo "  GFS_UTIL does not support JET "
+   echo "  "
+   exit
+fi
+
```

Comment:
don't need this block.  please remove.
We don't check for every machine here, so no need to do check for jet here.